compiler/java/src/test/java/com/c/LanguageDescription.java

```diff
@@ -0,0 +1,32 @@
+package com.c;
+
+public class LanguageDescription {
```

Comment:
That is a very smart and clean approach here. Definetly gonna use something like this in the C++ implementation as well!